### PR TITLE
Rag pipeline documentation

### PR DESCRIPTION
**Pull request by estohlmann**
Rag Pipeline Documentation

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/user-interface/react/src/shared/modal/confirmation-modal.tsx [hunk 1]
```diff
@@ -18,14 +18,13 @@ import { Modal as CloudscapeModal, Box, SpaceBetween, Button } from '@cloudscape
...
 import { dismissModal } from '../reducers/modal.reducer';
-import { MutationActionCreatorResult } from '@reduxjs/toolkit/query';
 
 export type CallbackFunction<T = any, R = void> = (props?: T) => R;
 
 export type ConfirmationModalProps = {
     action: string;
     resourceName: string;
-    onConfirm: () =>  MutationActionCreatorResult<any>;
+    onConfirm: () =>  any;
```
Comment (dustins) [inline lib/user-interface/react/src/shared/modal/confirmation-modal.tsx:27]: Are we doing anything with the result? Should it be `void` instead of `any`?

Comment (estohlmann) [reply lib/user-interface/react/src/shared/modal/confirmation-modal.tsx:27]: Trying to keep it future safe incase anything ever does anything with the response, but I could rip it out with our current use cases I suppose.

File: lib/docs/user/rag.md [hunk 2]
```diff
@@ -1 +1,64 @@
-# TODO
+# Automated Document Vector Store Ingestion Pipeline
+
+## Overview
+The Automated Document Ingestion Pipeline is designed to enhance LISA's RAG capabilities. This feature provides customers with a flexible, scalable solution for loading documents into configured vector stores. Customers have two methods to load files into vector stores configure with LISA. Customer can either manually load files via the chatbot user interface (UI), or via an ingestion pipeline. Files loaded via the chatbot UI are limited by Lambda's service limits on document file size and volume. Documents loaded via a pipeline are not subject to these limits, further expanding LISA’s ingestion capabilities. This pipeline feature supports the following document file types: PDF, docx, and plain text. The individual file size limit is 50 MB.
```
Comment (dustins) [inline lib/docs/user/rag.md:4]: - Customers have two methods to load files into vector stores ~~configure~~ **configured** with LISA.
- ~~Customer~~ **Customers** can either manually load files via the ch

File: lib/docs/user/rag.md [hunk 3]
```diff
@@ -1 +1,64 @@
...
+
+Customers can set up many ingestion pipelines. For each pipeline, they define the vector store and embedding model, and ingestion trigger. Each pipeline can be set up to to run based on an event trigger, or to run daily. From there, pre-processing kicks off to convert files into the necessary format. From there, processing kicks off to ingest the files with the specified embedding model and loads the data into the designated vector store. This feature leverages LISA’s existing chucking and vectorizing capabilities.
```
Comment (dustins) [inline lib/docs/user/rag.md:6]: - set up to ~~to~~ run
- This feature leverages LISA’s existing ~~chucking~~ **chunking** and vectorizing capabilities.

**Review by dustins — APPROVED** [review]

**Pull request merged**